common.blocks/button/button.bemhtml

```diff
@@ -1,4 +1,11 @@
 block('button')(
+    def()(function() {
+        var isRealButton = (['button', 'input'].indexOf(apply('tag')) >= 0) &&
```

Comment:
1) in this case you should change `def` mode(not `tag`) in attach too. And it's very strange to change tag in `def` mode .
2) agree